modules/installing-oadp-rosa-sts.adoc

```diff
@@ -11,7 +11,17 @@ AWS Security Token Service (AWS STS) is a global web service that provides short
 
 [IMPORTANT]
 ====
-Restic and Kopia are not supported in the OADP on ROSA with {aws-short} {sts-short} environment. Verify that the Restic and Kopia node agent is disabled.
+Restic is unsupported.
+
+Kopia file system backup (FSB) is supported when backing up file systems that do not have Container Storage Interface (CSI) snapshotting support.
+
+Example file systems include the following:
```

Comment:
Nit. This should be rephrased [to conform with the ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#following). As a lead-in for a list or procedure, _following_ should be used as an adjective and not a noun:
```suggestion
Supported file systems include the following examples:
```
That said, this isn't a merge blocker. Please take care of this in the future in a follow up PR.